webview-ui/src/components/settings/ApiOptions.tsx

```diff
@@ -1350,6 +1354,35 @@ const ApiOptions = ({
 				</>
 			)}
 
+			{selectedProvider === "fireworks" && (
```

Comment:
Can we put this as the second item in the list below Kilo Code?

Reply:
done!